### DISCUSSION TITLE
[FR]: Card View presentation

### DISCUSSION THREAD

**Discussion by asterox** [RafaelGB/obsidian-db-folder#802] [Ideas]
Hello.
I'd love to be able to visualize the db in cards set up in columns based on some parameter. Something like Kanban but with more flexibility: I'd like to be able to see some of the note content in the card for example for quick scanning.
Thank you!

**Comment by RafaelGB** [comment]
Hello @asterox ,

In the short to medium term I do not plan to implement new views. I would like to stabilize the application before reaching a point of equilibrium and already propose this type of additions

**Comment by asterox** [comment]
understood. ty!
